### PR TITLE
Setup Lists Layout

### PR DESCRIPTION
**Pull request by ksdenisova**
**As** a user
**I want** to see all lists titles
**So that** it allows to work with lists (choose list, add a new list and delete list)

**Given:** To Doit App
**When:** The app is displayed
**Then:** all lists titles has displayed depends on a new layout and acceptance criteria

**Acceptance:**
- different font size and color
- change add item image to MUI icon
- a new item text box on the down of list

[Trello](https://trello.com/c/yqXLCbfA/3-set-up-layout-for-displaying-lists-titles)

Current design:
image

Expected design:
image

**Refactoring:**
- Rename `ToDoit` to `Home`
- move main css to root css
- change add image to icon

### CODE REVIEW

File: to-do-app/src/components/Home/index.test.js [hunk 1]
```diff
@@ -81,66 +63,40 @@ test("renders submit button", () => {
...
 
   await act(async () => {
-    render(<ToDoit />);
+    render(<Home />);
   });
 
   expect(wrapper.find('button').prop('disabled')).toEqual(true);
 
-  wrapper.find('ul').first().simulate('click');
+  wrapper.setState({ activeList: "some list" })
```
Comment (grace000) [inline to-do-app/src/components/Home/index.test.js:95]: Nice!

File: to-do-app/src/components/ListTitles/index.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import NewList from '../NewList';
+import '../../ToDoApp.css';
+import './style.css';
+import React, { Component } from 'react';
+import CircleOutlinedIcon from '@mui/icons-material/CircleOutlined';
+
+class ListTitles extends Component {
+  filterLists(title) {
```
Comment (grace000) [inline to-do-app/src/components/ListTitles/index.js]: This function is filtering for active list only, right? I would rename to something like getActiveList.

File: to-do-app/src/components/ListTitles/index.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    return activeList[0];
+  }
+
+  getTitles() {
+    return this.props.toDoLists.map(entry => entry.title);
+  }
+
+  render() {
+    var lists = <ul></ul>;
```
Comment (grace000) [inline to-do-app/src/components/ListTitles/index.js]: Why the use of `var` here?

File: to-do-app/src/components/ListTitles/index.js (hunk 2 above)
Comment (ksdenisova) [reply to-do-app/src/components/ListTitles/index.js]: Yes, I was thinking about this too. I'll rename it.

File: to-do-app/src/components/Home/index.test.js (hunk 1 above)
Comment (ksdenisova) [reply to-do-app/src/components/Home/index.test.js:95]: Thanks!

File: to-do-app/src/components/ListTitles/index.js (hunk 3 above)
Comment (ksdenisova) [reply to-do-app/src/components/ListTitles/index.js]: No reason, I'll change it to `let`.

**Pull request merged**